apiserver/backrestservice/backrestimpl.go

```diff
@@ -125,6 +125,14 @@ func CreateBackup(request *msgs.CreateBackrestBackupRequest, ns, pgouser string)
 			return resp
 		}
 
+		//check if the current cluster is not upgraded to the deployed
+		//Operator version. If not, do not allow the command to complete
+		if cluster.Spec.UserLabels[config.LABEL_IS_UPGRADED] == msgs.False {
```

Comment:
I can work on that update, but, just to make sure I'm understanding correctly, `msgs.False` would instead be in `$PGOROOT/config/labels.go` as `UserLabelFalse` with a corresponding `UserLabelTrue`?

for the last question, agreed that this should now be an annotation (as well as the `upgrade-info` label that stores which version the cluster was upgraded from and to). I'll work on that change now.

Reply:
So a few things seemed to have changed since my original comment, so I'll answer it like this:

Now that it's in the annotation, we shoudl put something into `config/annotations.go` like `ANNOTATIONS_FALSE` (which...well I have a lot of feelings about that but that at least keeps it consistent)

and perhaps from there that will keep it...clean.

Reply:
Sounds like a good plan. I've removed the value from `common.go` and added it to `annotations.go`. It's only ever used in conjunction with a check on upgrade status, which I think keeps things 'cleaner' for the time being. I'll plan to stop at that unless you'd recommend continuing to modify more of the code base.